modules/serverless-event-delivery-parameters.adoc

```diff
@@ -6,14 +6,12 @@
 [id="serverless-event-delivery-parameters_{context}"]
 = Configurable event delivery parameters
 
-Knative Eventing provides configuration parameters that can be used to control what happens to events in cases where events fail to be delivered. For example, you can configure Knative to retry sending events that failed to be consumed, or forward these events to a dead letter sink.
-
 The following parameters can be configured for event delivery:
 
-Dead letter sink:: You can configure the `deadLetterSink` delivery parameter so that if an event fails to be delivered it is sent to the specified event sink.
+Dead letter sink:: You can configure the `deadLetterSink` delivery parameter so that if an event fails to be delivered, it is stored in the specified event sink. Undelivered events that are not stored in a dead letter sink are dropped. The dead letter sink be any Addressable object that conforms to the Knative Eventing sink contract, such as a Knative service, a Kubernetes service, or a URI.
```

Comment:
Just a small question, does Addressable have to be capitalized here? 
Ignore if its right!

Reply:
Probably not, I think I just didn't change it because it was here before, but I can update it 🙂